### ISSUE TITLE
Game stutter and frame-freezing

### ISSUE THREAD

**Issue by Asphelite** [R2Northstar/Northstar#415]
### Bug
The game stutters and keeps microfreezing during regular Northstar sessions. It is not tied to the server you are on, it seems to be different in severity depending on the map youre playing on. 
It kind of ruins the fun entirely 😅 

### Steps to reproduce

  1. Just join any server really, I have encountered numerous reports in the in-game chat by now.

### Specifications
  - Northstar version: 1.11.4
  - Platform: Origin

**Comment by Asphelite** [comment]
Very sorry that I cant provide any logs or dmps, but there isnt really anything that I could think of that would be of help here. It definitely looks like a problem with the Northstar-Client and its way of processing Server-Log Ins, etc.

**Comment by anticode-403** [comment]
I've had a very similar issue since installing Northstar a few weeks ago.

My fix:
 - Fullscreen
 - Enable Triple Buffer in-game
 - Open NVidia Control Panel and disable V-Sync and Triple Buffer
 - Launch the game in admin.

There are still a number of micro-stutters, and there are good and bad days for it even with this fix, but it does massively help.

**Comment by GeckoEidechse** [comment]
> Launch the game in admin.

Note that launching the game as admin really isn't recommended. Any exploit found in the game can do a lot more damage when you run the game as admin as opposed to normal user.

**Comment by anticode-403** [comment]
True, but it was suggested as a fix and it kind of, sorta helped, ish, at least a bit.

**Comment by Asphelite** [comment]
> * Fullscreen
> * Enable Triple Buffer in-game

Triple Buffering seemed to have done the trick for now, mind you on this was done on the new v1.12. Will test more later and close the issue if applicable. 

I think this should be mentioned in a FAQ!
